src/main/lrsql/init/customization.clj

```diff
@@ -0,0 +1,15 @@
+(ns lrsql.init.customization
```

Comment:
I would suggest making this namespace (and the corresponding resource dir) "localization" or something like that, to differentiate between general customization in the release

Reply:
In that scheme the file in the release would be like `/customization/lrsql/localization/language.json` and the classpath arg would be `-cp customization`

Reply:
Something like the FE index doc would be `/customization/public/admin/index.html` could be included with the same classpath override